libs/features/mas/src/variants/plans.js

```diff
@@ -17,7 +17,7 @@ export const PLANS_AEM_FRAGMENT_MAPPING = {
   allowedBadgeColors: ['spectrum-yellow-300-plans', 'spectrum-gray-300-plans', 'spectrum-gray-700-plans', 'spectrum-green-900-plans'],
   allowedBorderColors: ['spectrum-yellow-300-plans', 'spectrum-gray-300-plans'],
   borderColor: { attribute: 'border-color' },
-  size: ['wide', 'super-wide'],
+  size: ['wide', 'super-wide', 'students'],
```

Comment:
What if there is a wide Student card? We should not misuse fields.

Reply:
we add 'studebts-wide' option then